lib/similarity_utils.py

```diff
@@ -262,3 +263,17 @@ def similarity_one_vs_all_new(data: pd.DataFrame, main_actor: List[Any], reduced
     for actor in actors:
         similarities.append(similarity_new(main_actor, actor, reduced_dataset))
     return list(data["nconst"]), similarities
+
+def get_actor_data(train_data: pd.DataFrame, validate_data: pd.DataFrame, actor_id: str) -> List[Any]:
```

Comment:
to już prawie na pewno czepianie się, ale w kodzie pomiędzy metodami powinny być 2 wolne linijki. też przyjemniej się na to patrzy (przynajmniej mi)

Reply:
dodałam